### PR TITLE
CLI: dry-run dir warn; show disabled registry

### LINKED ISSUE #18768: quarkus registry list not reporting correctly
### Describe the bug

given a `~/.quarkus/config.yaml` with 

```
debug: true
registries:
  - playground.io:
      disabled: true
      update-policy: "always"
      maven:
        repository:
          url: http://localhost:8080/registry/maven
```

`quarkus registry list` says:

```
Available Quarkus extension registries:
- registry.quarkus.io
(Read from /Users/max/.quarkus/config.yaml)
```

it should have been

```
Available Quarkus extension registries:
- playground.io
(Read from /Users/max/.quarkus/config.yaml)
```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### LINKED ISSUE #18773: quarkus create --dry-run test fails if test dir already exist
### Describe the bug

`quarkus create --dryrun  test` when you already have a directory called test fails with error.

It is a dry-run so should it not just warn and still print out what it was planning to do ?

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ebullient**
Resolves #18773 
Resolves #18768

### CODE REVIEW

File: devtools/cli/src/main/java/io/quarkus/cli/RegistryListCommand.java [hunk 1]
```diff
@@ -19,14 +20,15 @@
     public Integer call() throws Exception {
 
         registryClient.refreshRegistryCache(output);
+        final RegistriesConfig config = RegistriesConfigLocator.resolveConfig();
 
         output.info("Available Quarkus extension registries:");
```
Comment (aloubyansky) [inline devtools/cli/src/main/java/io/quarkus/cli/RegistryListCommand.java:25]: If it is including disabled registries, maybe it shouldn't start with `Available`? That was the point of this method - to show what's enabled.

Comment (aloubyansky) [reply devtools/cli/src/main/java/io/quarkus/cli/RegistryListCommand.java:25]: I kind of still think it's a good idea, btw.

**Comment by gsmet** [comment]
Should this be backported to 2.1?

**Comment by aloubyansky** [comment]
Yes, I am reviewing something here.

**Review by aloubyansky — APPROVED** [review]

**Pull request merged**